src/sklearn_evaluation/plot/classification.py

```diff
@@ -30,9 +31,11 @@ def plot(self, ax=None):
         if ax is None:
             _, ax = plt.subplots()
 
+        cmap = default_cmap()
+
         _plot_cm(
             self.cm,
-            cmap=default_heatmap(),
```

Comment:
nit: I think we can keep this 
```
        _plot_cm(
            self.cm,
            cmap=default_heatmap(),
```

Reply:
@yafim please address

Reply:
We updated the default cmap with new colors, so using this function will return the old coloring. I kept this function since we might want to go back to the old coloring or toggle between them.